src/integrationTest/java/org/opensearch/security/TelemetryPluginIntegrationTests.java

```diff
@@ -0,0 +1,39 @@
+package org.opensearch.security;
+
+import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.opensearch.test.framework.TestSecurityConfig;
+import org.opensearch.test.framework.cluster.ClusterManager;
+import org.opensearch.test.framework.cluster.LocalCluster;
+import org.opensearch.test.framework.cluster.TestRestClient;
+
+import org.opensearch.telemetry.OTelTelemetryPlugin;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.opensearch.test.framework.TestSecurityConfig.AuthcDomain.*;
+import static org.opensearch.test.framework.TestSecurityConfig.Role.ALL_ACCESS;
+
+@RunWith(com.carrotsearch.randomizedtesting.RandomizedRunner.class)
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
+public class TelemetryPluginIntegrationTests {
```

Comment:
Can you please provide info about why this needs to be in the Security plugin? 

I understand that the Telemetry plugin was added to core but this means that a change in the Telemetry plugin could cause a test in Security to fail--it makes it upstream of Security. 

I don't think we should have any plugins upstream of Security if we can help it and I am not sure why this test needs to be housed in the Security repo instead of using the same framework other core-internal plugins (S3 for example) do.

Reply:
Tagging @Gaganjuneja to answer this since he created the issue and suggested to write the test in security repo.

Reply:
@Gaganjuneja @atharvasharma61 If the intent is to test the plugin behavior with security, I'd suggest to write a sanity test inside telemetry plugin repo with security plugin enabled